commands/command_uninit.go

```diff
@@ -20,7 +22,7 @@ var (
 )
 
 func uninitCommand(cmd *cobra.Command, args []string) {
-	if err := lfs.UninstallFilters(); err != nil {
+	if err := lfs.TeardownFilters(); err != nil {
 		Error(err.Error())
```

Comment:
Why the name change from install/uninstall to setup/teardown? The `Hook` type still uses the install/uninstall/upgrade terms.